examples/all-clusters-app/linux/main-common.cpp

```diff
@@ -336,3 +337,29 @@ void emberAfThermostatClusterInitCallback(EndpointId endpoint)
 
     SetDefaultDelegate(endpoint, &delegate);
 }
+
+using namespace chip::app;
+chip::Protocols::InteractionModel::Status emberAfExternalAttributeReadCallback(EndpointId endpoint, ClusterId clusterId,
+                                                                               const EmberAfAttributeMetadata * attributeMetadata,
+                                                                               uint8_t * buffer, uint16_t maxReadLength)
+{
+
+    VerifyOrReturnValue(clusterId == Clusters::UnitTesting::Id, chip::Protocols::InteractionModel::Status::Failure);
+    VerifyOrReturnValue(attributeMetadata != nullptr, chip::Protocols::InteractionModel::Status::Failure);
```

Comment:
This function could use a `using chip::Protocols::InteractionModel::Status` at the top and `Status::Failure` etc after that.